gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcWriteChannelTest.java

```diff
@@ -360,16 +359,6 @@ public void writeHandlesErrorOnInsertRequestWithoutUncommittedData() throws Exce
     assertThrows(IOException.class, writeChannel::close);
   }
 
-  @Test
-  public void writeHandlesErrorOnStartRequestFailure() throws Exception {
```

Comment:
We should modify the test instead of deleting it, let's try to assert on the write/close method for this scenario

Reply:
This test was for a a configuration value which we no longer use